certified-connectors/Muhimbi PDF Converter Online/apiProperties.json

```diff
@@ -1,36 +1,34 @@
-{
-  "properties": {
-    "connectionParameters": {
-      "token": {
-        "type": "oauthSetting",
-        "oAuthSettings": {
-          "identityProvider": "oauth2",
-          "clientId": "001E677C-8056-47B1-B415-A461F2230223",
-          "scopes": [
-            "Flow"
-          ],
-          "redirectMode": "Global",
-          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
-          "properties": {
-            "IsFirstParty": "False"
-          },
-          "customParameters": {
-            "authorizationUrl": {
-              "value": "https://api.muhimbi.com/auth/pages/authorize.aspx"
-            },
-            "tokenUrl": {
-              "value": "https://api.muhimbi.com/auth/pages/gettoken.aspx?grant_type=authorization_code"
-            },
-            "refreshUrl": {
-              "value": "https://api.muhimbi.com/auth/pages/gettoken.aspx?grant_type=refresh_token"
-            }
-          }
-        }
-      }
-    },
-    "iconBrandColor": "#2B4D5A",
-    "capabilities": [],
-    "publisher": "Muhimbi Ltd",
-	"stackOwner": "Muhimbi Ltd"
-  }
-}
\ No newline at end of file
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "001E677C-8056-47B1-B415-A461F2230223",

```

Comment:
@mmohapatra-muhimbi secret look alike values are not allowed. please replace with dummy value.

Reply:
Its the client ID not the client secret, I checked previous commits also it was there

Reply:
@mmohapatra-muhimbi it might be missed from previous PRs, you can replace with dummy value in GitHub and you can provide us with the value in ISV Studio.

Reply:
@Amjed-Ayoub updated clientid to dummy guid